removal_from_ansible.rst

```diff
@@ -125,8 +125,16 @@ Collections that have been announced deprecated / unmaintained by their maintain
 Process
 ~~~~~~~
 
-#. `Announce upcoming removal from the next major Ansible release <announce_removal_>`_.
-#. `Remove collection from the next major Ansible release <remove_collection_>`_.
+If the latest stable Ansible version is X.something:
+
+#. Announce upcoming removal in the Ansible changelog (``https://github.com/ansible-community/ansible-build-data/blob/main/<X>/changelog.yaml``).
+#. Remove from ``ansible.in`` (``https://github.com/ansible-community/ansible-build-data/blob/main/<X+1>/ansible.in``).
+#. Remove from ``collection-meta.yaml`` (``https://github.com/ansible-community/ansible-build-data/blob/main/<X+1>/collection-meta.yaml``).
+#. Document actual removal for the next Ansible release in the changelog (``https://github.com/ansible-community/ansible-build-data/blob/main/<X+1>/changelog.yaml``).
```

Comment:
The goal of referring to the general processes inside this document was to avoid repeating these instructions everywhere.

Reply:
I understand, but referring to `announce removal` and `remove collection` looks like there are two separate steps. I think we can do it in just one step. The process I propose is not simply `announce removal` + `remove collection` ;-)

Additionally, `Announcing upcoming removal` says to `announce upcoming removal in the collection's issue tracker`. Which doesn't make sense if the collection is officially unmaintained. And, actually, wouldn't even be possible if the collection has been archived.

Reply:
I don't see the point of two divergent processes for doing the exact same thing, or duplicating documentation details in order to do so. If you think that in some circumstances announcing it on the collection's issue tracker won't be possible, just add an "if possible" on the end of the existing documentation for that step.

Reply:
@felixfontein @flowerysong What do you think about my latest commit? Does it look OK to you now?